test_github_workflows.py

```
@@ -0,0 +1,793 @@
+                    {'uses': 'actions/checkout@v3'},
+                    {'run': f'echo "Job {i}"'}
+                ]
+            }
+        
+        # Test parsing performance
+        import time
+        start_time = time.time()
+        yaml_content = yaml.dump(large_workflow)
+        parsed = yaml.safe_load(yaml_content)
+        end_time = time.time()
+        
```

Flake8 (7.2.0)

[warning] blank line contains whitespace

(W293)